tools/cli/clitest/context.go

```diff
@@ -61,6 +62,15 @@ func BoolArgument(name string, value bool) CliArgument {
 	}
 }
 
+// StringSliceArgument introduces a new string slice argument for cli context
+func StringSliceArgument(name string, values ...string) CliArgument {
```

Comment:
I suggest adding this in the clitest test case :)

Reply:
Added!